src/main/java/com/sngular/kloadgen/processor/objectcreatorfactory/impl/AvroObjectCreatorFactory.java

```diff
@@ -118,6 +127,7 @@ public final Object createValueObject(final SchemaProcessorPOJO pojo) {
   public final void assignRecord(final SchemaProcessorPOJO pojo) {
     final GenericRecord entityObject = entity.get(pojo.getRootFieldName());
     entityObject.put(pojo.getFieldNameSubEntity(), entity.get(pojo.getFieldNameSubEntity()));
+    entity.get(pojo.getRootFieldName());
   }
```

Comment:
Is this function doing anything?
It seems like the last line may have been intended to be `entity.put(pojo.getRootFieldName(), entityObject);`

Reply:
Done